specification/iotsecurity/resource-manager/readme.md

```diff
@@ -28,6 +28,18 @@ directive:
     from: defenderSettings.json
     where: '$.paths["/subscriptions/{subscriptionId}/providers/Microsoft.IoTSecurity/defenderSettings"].get'
     reason: The list returns limited number of items
+  - suppress: PageableOperation
+    from: sensors.json
+    where: '$.paths["/{scope}/providers/Microsoft.IoTSecurity/sensors/{sensorName}"].get'
```

Comment:
So, these suppressions on pageable are a bit dubious.  It is often the case that limited numbers of resources get larger, and then there is a breaking change to support paging.  It is better to future-proof this with paging now.

Reply:
Unless there is some concrete reason why this list can *never* be large enough for paging.